content/docs/guides/neon-identity.md

```diff
@@ -0,0 +1,149 @@
+---
+title: About Neon Identity
+subtitle: Automatically sync user profiles from your auth provider directly to your database
+enableTableOfContents: true
+---
+
+_<b>Q:</b> This EAP message is re-purposed from another feature. We can modify if, for example, we want to link to sign up for the EAP etc_
+
+<ComingSoon />
+
+**Neon Identity** connects your authentication provider to your Neon database, automatically synchronizing user profiles. Access your user data directly in your database environment, with no custom integration code needed.
+
+## Authentication and synchronization
+
+When implementing user authentication, it is common to use dedicated authentication providers like StackAuth, Clerk, and others to handle the complexities of user identity, passwords, and security. However, keeping your database in sync with these providers typically requires additional development work.
+
+Neon Identity solves this by maintaining a direct connection between your auth provider and database, ensuring your application always has access to up-to-date user information right from your database.
```

Comment:
"solves this by integrating your auth provider with your Postgres database" or something similar, I think that "by maintaining a direct connection" seems to be related to websockets, at least that's what I get from reading it